packages/central-server/app/tasks/PatientMergeMaintainer.js

```diff
@@ -206,6 +208,47 @@ export class PatientMergeMaintainer extends ScheduledTask {
     return records;
   }
 
+  async specificUpdate_PatientProgramRegistration() {
+    const { PatientProgramRegistration } = this.models;
+    const patientProgramRegistrationMerges = await this.findPendingMergePatients(
```

Comment:
Unrelated to your changes, but I'm confused about the `findPendingMergePatients` util. It seems to find _all_ merged patients, not just those still requiring child records to be handled?

Reply:
Yeah true, so it doesn't find any child records for the patients that are already handled. But it still searches for them! It would be nice to improve that. Fine to call it out of scope here if you like.